CMakeLists.txt

```diff
@@ -1350,6 +1352,25 @@ if(MSVC)
   )
 endif()
 
+if(WIN32)
+  find_file(QT_TRANSLATION_FILE qt_de.qm PATH_SUFFIXES share/qt5/translations)
```

Comment:
Please add a comment what this block is about. Does it install QT's translations?

Reply:
obviously...

Reply:
I'll add a comment. `qt_de.qm` is just one arbitrary file in that directory.